src/components/MessageCard/components/MessageCard.Survey.Thumbs.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { SurveyContext } from '../MessageCard.Survey.context'
+import { EmojiButtonUI, SurveyOptionsUI } from './MessageCard.Survey.css'
+
+const buttons = [
+  { id: 'thumbs-up', text: '👍' },
+  { id: 'thumbs-down', text: '👎' },
+]
+
+export const MessageCardSurveyThumbs = () => {
+  const { onSelection = () => {}, selected = null } = React.useContext(
```

Comment:
I think there is `noop` function in utils that can be used here :)

And I am thinking, for the future we could extract it to custom hook and provide default values there, so that any next components that would be using that, wouldn't need to worry about defaults. What do you think?

Reply:
Good call!